modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractTypeScriptClientCodegen.java

```diff
@@ -236,6 +235,8 @@ protected String nullableQuotedJSString(@Nullable String string) {
     public static final String NULL_SAFE_ADDITIONAL_PROPS = "nullSafeAdditionalProps";
     public static final String NULL_SAFE_ADDITIONAL_PROPS_DESC = "Set to make additional properties types declare that their indexer may return undefined";
 
+    public static final String LICENSE_NAME_DEFAULT_VALUE = "Unlicense";
```

Comment:
i think this should be empty by default

```suggestion
    public static final String LICENSE_NAME_DEFAULT_VALUE = "";
```

Reply:
actually, since that seems to have been the default, lets keep this existing default.

Reply:
It is an odd default though, I agree, I was thrown by the same thing. And then the jQuery one with MIT seems even worse as a default, as it's way more permissive.